config/validate_server.go

```diff
@@ -20,10 +19,14 @@ func (s *ServerConfig) Validate() error {
 		return errors.New("Write timeout needs to be set for server configuration")
 	}
 
+	if s.Port < 10 || s.Port > 999999 {
```

Comment:
Still out of bounds:
max port number is 65 535

Reply:
You're right, changed